requirements.txt

```
@@ -1,17 +1,17 @@
-attrs==21.4.0  #  can't upgrade beyond that b/c chalice 1.27.3+9 depends on attrs<21.5.0 and >=19.3.0
+attrs==23.1.0
```

```suggestion
attrs==22.2.0
```

Later versions don't play nicely with PyCharm. I get 

```
"Expected type 'AttrsInstance', got '…' instead"
```

when calling `attrs.asdict`